test_elasticsearch/test_server/test_helpers.py

```diff
@@ -184,26 +184,44 @@ def test_errors_are_collected_properly(self):
 
 
 class TestScan(ElasticsearchTestCase):
-    def test_order_can_be_preserved(self):
+    def setUp(self):
+        super(TestScan, self).setUp()
         bulk = []
         for x in range(100):
             bulk.append({"index": {"_index": "test_index", "_type": "answers", "_id": x}})
             bulk.append({"answer": x, "correct": x == 42})
         self.client.bulk(bulk, refresh=True)
 
+    def test_order_can_be_preserved(self):
         docs = list(helpers.scan(self.client, index="test_index", doc_type="answers", size=2, query={"sort": ["answer"]}, preserve_order=True))
 
         self.assertEquals(100, len(docs))
         self.assertEquals(list(map(str, range(100))), list(d['_id'] for d in docs))
         self.assertEquals(list(range(100)), list(d['_source']['answer'] for d in docs))
 
-    def test_all_documents_are_read(self):
-        bulk = []
-        for x in range(100):
-            bulk.append({"index": {"_index": "test_index", "_type": "answers", "_id": x}})
-            bulk.append({"answer": x, "correct": x == 42})
-        self.client.bulk(bulk, refresh=True)
+    def test_general_kwargs_forwarded_to_search(self):
+        inexistent_index = 'test_index_123'
+        self.assertRaises(
+            NotFoundError,
+            lambda: list(helpers.scan(self.client, index=inexistent_index, doc_type="answers", size=2))
+        )
+        global_kwargs = {'ignore': 404}
+        list(helpers.scan(self.client, index=inexistent_index, doc_type="answers", size=2, global_kwargs=global_kwargs))
```

Comment:
it would be nice to verify the response here, even when the real test is to see no error was raised (or at least have a comment to that effect so readers know what is being tested)